contractsV5/KyberNetwork.sol

```diff
@@ -733,10 +738,10 @@ contract KyberNetwork is Withdrawable, Utils, IKyberNetwork, ReentrancyGuard {
         // Updates reserve eligibility and rebate percentages
         updateEligibilityAndRebates(eligibleWallets, rebatePercentages, tradeData);
 
-        // // Send total fee amount to fee handler with reserve data.
+        // Send total fee amount to fee handler with reserve data.
         require(
             feeHandlerContract.handleFees.value(tradeData.networkFeeWei)(eligibleWallets, rebatePercentages),
-            "Transfer network fee to FeeHandler failed"
+            "Fail fee trf (FeeHandler)"
```

Comment:
trf?
did u count num of chars?
any tool for this?

Reply:
trf = short form for transfer. I can stick to the full name length.

Yes I did. Copy paste into chrome console. Do "testString".length. Make sure <= 32.

Reply:
I like tx for transfer.
isn't that the common short term

I know we used it a lot in companies I worked for.
But not sure how spread it is?